src/components/pages/VendingMachine/MenuSelect.jsx

```diff
@@ -48,6 +50,12 @@ const MenuSelect = () => {
   const availableMilk =
     MENU_PRICE.milk <= coin && milk > 0 && cup > 0 ? false : true;
 
+  const availableCoke =
+    MENU_PRICE.coke <= coin && coke > 0 && cup > 0 ? false : true;
+
+  const availableSida =
+    MENU_PRICE.sida <= coin && sida > 0 && sida > 0 ? false : true;
```

Comment:
요거 잘하면 하나로 합칠수도 있겠네요! 아래 같은 느낌으로

```suggestion
  const isDisabledDrink = (drink) => MENU_PRICE[drink] > coin || menuCount[drink] <= 0 || cup <= 0;
```